### PR TITLE
proposal filter changes

### PR DESCRIPTION
**Pull request by julianjelfs**

### CODE REVIEW

**Review by megrogan — APPROVED** [review]

**Review by megrogan — COMMENTED** [review]
Needs translations

**Review by megrogan — CHANGES_REQUESTED** [review]
Needs translations

File: frontend/app/src/i18n/en.json [hunk 1]
```diff
@@ -593,7 +593,15 @@
         "disableAll": "Disable all",
         "readmore": "Read more",
         "readless": "Read less",
-        "details": "Details"
+        "details": "Details",
+        "builtInAction": "Built In Actions",
+        "userIndexAction": "User Index Actions",
+        "groupIndexAction": "Group Index Actions",
+        "notificationsAction": "Notifications Actions",
```
Comment (hpeebles) [inline frontend/app/src/i18n/en.json:600]: Should be 'Notifications Index Actions'

File: frontend/app/src/i18n/cn.json [hunk 2]
```diff
@@ -578,11 +578,11 @@
         "builtInAction": "内置动作",
         "userIndexAction": "用户索引操作",
         "groupIndexAction": "集团指数行动",
-        "notificationsAction": "通知操作",
         "proposalsBotAction": "提案机器人操作",
         "storageIndexAction": "存储索引操作",
         "cyclesDispenserAction": "循环分配器操作",
-        "unknownActionCategory": "其他行为"
+        "unknownActionCategory": "其他行为",
+        "notificationsAction": "通知索引操作"
```
Comment (megrogan) [inline frontend/app/src/i18n/cn.json:585]: Ideally the key would change too but no biggy.

File: frontend/app/src/i18n/es.json [hunk 3]
```diff
@@ -574,7 +574,15 @@
...
-        "details": "Detalles"
+        "details": "Detalles",
+        "builtInAction": "Acciones integradas",
+        "userIndexAction": "Acciones de índice de usuario",
+        "groupIndexAction": "Acciones de índice de grupo",
+        "proposalsBotAction": "Acciones del bot de propuestas",
+        "storageIndexAction": "Acciones de índice de almacenamiento",
+        "cyclesDispenserAction": "Acciones del dispensador de ciclos",
+        "unknownActionCategory": "Otras acciones",
+        "notificationsAction": "Índice de notificaciones Acciones"
```
Comment (hpeebles) [inline frontend/app/src/i18n/es.json:585]: I know zero Spanish... but this one doesn't fit the same pattern...

File: frontend/app/src/i18n/it.json [hunk 4]
```diff
@@ -573,7 +573,15 @@
...
         "readmore": "Leggi di più",
         "readless": "Leggi di meno",
-        "details": "Dettagli"
+        "details": "Dettagli",
+        "builtInAction": "Azioni incorporate",
+        "userIndexAction": "Azioni indice utenti",
+        "groupIndexAction": "Azioni indice di gruppo",
+        "proposalsBotAction": "Proposte Bot Azioni",
+        "storageIndexAction": "Azioni dell'indice di archiviazione",
+        "cyclesDispenserAction": "Cicli Azioni Dispenser",
```
Comment (hpeebles) [inline frontend/app/src/i18n/it.json:582]: This says 'Cycles Actions Dispenser'

**Pull request merged**